APIComparer.Backend.Tests/APIComparer.Backend.Tests.csproj

```diff
@@ -69,6 +85,9 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
This is some marker that VisualStudio automatically adds to identify test projects to only load the Test Explorer when that makes sense (see http://stackoverflow.com/a/22917902/335418).